app/views/admin/dashboard/show.html.erb

```diff
@@ -1 +1,16 @@
-<h1>Admin Dashboard</h1>
\ No newline at end of file
+<h1>Admin Dashboard</h1>
+
+<% if @innings && !@innings.empty? %>
```

Comment:
I don't think this if statement is necessary because there should always be a current inning. If there are no innings for some reason, then something has gone wrong. So it would actually be preferable to display the header with an empty list to clue an admin into the fact that something has gone wrong.

Reply:
This is a really good point, we saw this logic in the user innings view and carried it over.  But you are correct, if there are no innings to display something is wrong.